### ISSUE TITLE
Deno Module Page Showing "No uploaded versions" Error

### ISSUE THREAD

**Issue by CodingCarter** [denoland/deno_website2#1464]
I uploaded my Deno [Minifier module](https://github.com/CodingCarter/minifier) to the deno.land's third party modules and after going through all of the designated steps, I went to the page at [https://deno.land/x/minifier](https://deno.land/x/minifier) and the website is showing this
![image](https://user-images.githubusercontent.com/43641633/90035645-a713a680-dc87-11ea-8c5b-3be9820ce7fb.png)

I don't really understand why this is happening, as I have two versions uploaded on the [github repo releases](https://github.com/CodingCarter/minifier/releases).  Can someone please help me resolve this error.

**Comment by ry** [comment]
@CodingCarter We made a big refactor to how the deno.land/x website works to increase stability. Unfortunately that has caused a bit of instability like you're experiencing as we move over to the new system. Read about it here https://deno.land/posts/registry2

TLDR: go to https://deno.land/x#add and put in "minifier" click next even tho it's grayed out.

**Comment by lucacasonato** [comment]
@CodingCarter I see you have gone through all of the steps to add it. It is showing up for me. I think it is just caching some old data on your side. Could you hard refresh to wipe the cache? (Ctrl-Shift-R) on Chrome.

**Comment by CodingCarter** [comment]
> @CodingCarter I see you have gone through all of the steps to add it. It is showing up for me. I think it is just caching some old data on your side. Could you hard refresh to wipe the cache? (Ctrl-Shift-R on Chrome)

Thank you very much, this fixed my issue 👍
